dbms/src/Storages/Transaction/TMTContext.h

```diff
@@ -140,8 +139,8 @@ class TMTContext : private boost::noncopyable
 
     std::atomic_uint64_t replica_read_max_thread;
     std::atomic_uint64_t batch_read_index_timeout_ms;
-    std::atomic_uint64_t wait_index_timeout_ms;
-    std::atomic_uint64_t read_index_worker_tick_ms;
+    std::atomic_uint64_t wait_index_timeout_ms{};
+    std::atomic_uint64_t read_index_worker_tick_ms{};
     std::atomic_int64_t wait_region_ready_timeout_sec;
```

Comment:
better to init these variables in the ctor of `TMTContext`. It is weird that some atomic value is inited in here, but some of them are inited in the ctor.

https://github.com/pingcap/tiflash/blob/488a64c72eaadc6ece8e266774e5a47389c8ad31/dbms/src/Storages/Transaction/TMTContext.cpp#L39-L59

Reply:
And set the default value by `wait_index_timeout_ms` to be `DEFAULT_WAIT_INDEX_TIMEOUT_MS`, `read_index_worker_tick_ms` to be `10` (better add a constant name for `10`)